core/shared/src/main/scala/scalaz/zio/duration/Duration.scala

```diff
@@ -60,6 +63,9 @@ sealed trait Duration extends Ordered[Duration] with Serializable with Product {
 
   /** The `java.time.Duration` returned for an infinite Duration is technically "only" ~2x10^16 hours long (`Long.MaxValue` number of seconds) */
   def asJava: JavaDuration
+
+  final def relative: Delay = Delay.relative(this)
+  final def absolute: Delay = Delay.absolute(toMillis)
```

Comment:
Does `absolute` really make sense on `Duration`???

Reply:
You're right. Doesn't make sense. I'll remove it